FetchRequests/Sources/Controller/CWFetchedResultsController.swift

```diff
@@ -906,24 +891,27 @@ private extension CWFetchedResultsController {
         }
 
         for (index, sort) in sortDescriptors.enumerated() {
+            guard let keyPath = sort.key, keyPath != "self" else {
```

Comment:
Where does the `self` thing come from?

Reply:
Oh good catch. I had played with some alternatives for responding to "id" that I ripped out but I think I want to put back.

Reply:
Pushed the relevant change